doc/classes/Node.xml

```diff
@@ -905,6 +905,10 @@
 		</method>
 	</methods>
 	<members>
+		<member name="auto_translate_mode" type="int" setter="set_auto_translate_mode" getter="get_auto_translate_mode" enum="Node.AutoTranslateMode" default="0">
+			Defines if any text should automatically change to its translated version depending on the current locale (for nodes such as [Label], [RichTextLabel], [Window], etc). See [enum AutoTranslateMode].
```

Comment:
The voice in my head called Yuri told me to add the period to "etc" last time.
```suggestion
			Defines if any text should automatically change to its translated version depending on the current locale (for nodes such as [Label], [RichTextLabel], [Window], etc.). See [enum AutoTranslateMode].
```

Reply:
There are plenty of instances of non-punctuated "etc" in the docs (including this very file), but sure.